packages/tooltip/src/css/index.js

```diff
@@ -36,51 +41,87 @@ export default {
     color: core.colors.white,
     backgroundColor: core.colors.blue
   },
-  // --tailPosition
-  [`.psds-tooltip:after`]: {
+
+  [`.psds-tooltip__tail`]: {
```

Comment:
i'm seeing some clipping of the tail shadow - on the left

![screen shot 2019-01-04 at 9 05 26 am](https://user-images.githubusercontent.com/226356/50697619-f996d680-0fff-11e9-8f48-7e731d05fd2a.png)

i wonder if you can make the width a bit larger to give the shadow some room and center the arrow

Reply:
:done:, now there is a box shadow blur width on both sides of the overflow-hidden parent container.